.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  ignore:
+  - dependency-name: io.jenkins.plugins*
+    versions:
+    - ">= 0"
+  - dependency-name: org.jenkins-ci.main:jenkins-core
+    versions:
+    - ">= 0"
+  - dependency-name: org.jenkins-ci.plugins*
```

Comment:
So what is the point of running Dependabot at all here? Suggest deleting whole `ignore` block.

Reply:
Agreed. Not needed since we have BOM for the most of dependencies